src/app/models/Annotation.ts

```diff
@@ -0,0 +1,148 @@
+import { Injector } from "@angular/core";
+import {
+  AUDIO_EVENT_TAG,
+  AUDIO_RECORDING,
+  SHALLOW_AUDIO_EVENT,
+  SHALLOW_SITE,
+  TAG,
+} from "@baw-api/ServiceTokens";
+import { Id } from "@interfaces/apiInterfaces";
+import { BehaviorSubject, Observable } from "rxjs";
+import { flatMap, map } from "rxjs/operators";
+import { AbstractModel } from "./AbstractModel";
+import type { AudioEvent } from "./AudioEvent";
+import type { AudioEventTag } from "./AudioEventTag";
+import type { AudioRecording } from "./AudioRecording";
+import type { Site } from "./Site";
+import type { Tag } from "./Tag";
+
+export interface IAnnotation {
+  id: Id;
+  audioEvent: Observable<AudioEvent>;
+  audioEventTags: Observable<AudioEventTag[]>;
+  tags: Observable<Tag[]>;
+  audioRecording: Observable<AudioRecording>;
+  site: Observable<Site>;
+}
+
+export class Annotation extends AbstractModel implements IAnnotation {
+  public readonly kind: "Annotation" = "Annotation";
+  public id: Id;
+  private _audioEvent?: Observable<AudioEvent>;
```

Comment:
This is not a standalone model.

It should extend AudioEvent, it should not take an AudioEvent as a field.

An _annotation_ is an _audio event_ with one more tags.